web/main/models.py

```diff
@@ -3683,41 +3725,44 @@ def _delete_related_links_and_text_blocks(self):
             cls.objects.filter(id__in=ids).delete()
 
     @property
-    def sections(self):
-        return Section.objects.filter(casebook=self).filter(resource_type__isnull=True)
+    def sections(self) -> models.QuerySet[Section]:
+        return Section.objects.filter(casebook=self, resource_type__isnull=True)
 
     @property
-    def resources(self):
+    def resources(self) -> ContentNodeQuerySet:
         return ContentNode.objects.filter(casebook=self, resource_id__isnull=False)
```

Comment:
Same, I think this method is wrong now... Something else I hope to clean up soon :-)